pep-0690.rst

```diff
@@ -328,54 +587,59 @@ startup time and memory usage.
 
 The reference implementation shows small performance regressions in a few
 pyperformance benchmarks, but improvements in others. (TODO update with
-detailed data from 3.11 port of implementation.)
+detailed data from main-branch port of implementation.)
 
 
 How to Teach This
 =================
 
 Since the feature is opt-in, beginners should not encounter it by default.
-Documentation of the ``-L`` flag and ``PYTHONLAZYIMPORTS`` environment variable
-can clarify the behavior of lazy imports.
+Documentation of the ``-L`` flag and ``importlib.enable_lazy_imports()`` can
+clarify the behavior of lazy imports.
+
+The documentation should also clarify that opting into lazy imports is opting
+into an experimental non-standard semantics for Python imports, which could
```

Comment:
How about "provisional" instead of "experimental non-standard"?  I'm of mixed mind about that.

Reply:
I don't want to remove "non-standard" because I think that's the key point here. I'd sooner remove "experimental" -- to me "experimental" and "provisional" both suggest "might be removed," which I'm not sure is true, but is somewhat beside the point here anyway. The important point is that it's non-standard, and libraries aren't obligated to support it.

Reply:
+1 for removing "experimental" and keeping "non-standard".